fabric-convention-tags-v2/src/client/java/net/fabricmc/fabric/impl/tag/convention/ConventionLogWarningsClient.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.tag.convention;
+
+import java.util.List;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.client.resource.language.I18n;
+import net.minecraft.item.Item;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKeys;
+import net.minecraft.registry.tag.TagKey;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
+import net.fabricmc.loader.api.FabricLoader;
+
+public class ConventionLogWarningsClient implements ClientModInitializer {
+	public static final Logger LOGGER = LoggerFactory.getLogger(ConventionLogWarningsClient.class);
+
+	/**
+	 * A config option mainly for developers.
+	 * Logs out modded item tags that do not have translations when running on integrated server.
+	 * Defaults to SILENCED.
+	 */
+	private static final String LOG_UNTRANSLATED_WARNING_MODE = System.getProperty("fabric-tag-conventions-v2.missingTagTranslationWarning", LogWarningMode.SILENCED.name());
```

Comment:
I would change this field to have a type of `LogWarningMode`, will make the code bellow a lot cleaner as you wont need to keep doing the case insenstive string comparisions.

Reply:
I'm not sure what you mean by this? The LOG_UNTRANSLATED_WARNING_MODE is just what the user enters for the system property. I figured having the check be case insensitive would help reduce confusion if the user types "dev_verbose" instead of "DEV_VERBOSE" by accepting either casing.

Reply:
Case insenstive is good, I would would change this so there is a static method that gets the `LogWarningMode` from the system prop, and sets in this field. Its only a minor code cleanup thing, and doesnt change how it works.

Reply:
Ah I see what you mean. Added that now along with logging if user puts in an invalid entry for the property